src/composables/pnftInteractions.ts

```diff
@@ -1,16 +1,29 @@
 import { PNFT } from '@/common/helpers/types';
 import usePinata from '@/composables/pinata';
+import useWallet from '@/composables/wallet';
+const { isConnected, getWallet, getWalletAddress } = useWallet();
 
 
-export function generateTicketDetailLink(ticket: PNFT) {
-    /* Input: Takes in a ticket (pinata NFT metadata)
-       Output: link to ticket detail page using mintID or undefined (some tickets may not have mintID)
-    */
-       const ticket_detail_url_prefix = "/ticketdetail/"
-       const attr_key = "mintId"
-       let attr = ticket.metadata.keyvalues.hasOwnProperty(attr_key) ? ticket.metadata.keyvalues[attr_key] : undefined
-       return typeof attr != 'undefined' ? ticket_detail_url_prefix + attr : undefined
-  }
+export function formatTicketDetailLink(questionMintId: string, appUrl?: string)  {
+   /* Input: Takes in a question's mintId 
+      Output: link to ticket detail page using mintID or undefined (some tickets may not have mintID)
+   */
+      const appUrlPrefix =  typeof appUrl != undefined ? appUrl : ""
```

Comment:
can we make the default appUrl ("https://gmnh.vercel.app") for now, instead of "")?